src/main/java/mate.academy/service/impl/ProductServiceImpl.java

```diff
@@ -2,13 +2,19 @@
 
 import java.util.List;
 import java.util.stream.Collectors;
+import mate.academy.lib.Component;
+import mate.academy.lib.Inject;
 import mate.academy.model.Product;
 import mate.academy.service.FileReaderService;
 import mate.academy.service.ProductParser;
 import mate.academy.service.ProductService;
 
+@Component
 public class ProductServiceImpl implements ProductService {
+    @Inject
     private ProductParser productParser;
+
+    @Inject
     private FileReaderService fileReaderService;
```

Comment:
Similarly, 'fileReaderService' could be more informative. If it reads from a specific source or has a specific use case, reflect that in the name, like 'localFileReaderService' if it reads files from the local filesystem.

Reply:
ok, done